### ISSUE TITLE
[SF_ACC][Feature Request][FileUploader]: adding "required" & "accessibleName" property

### ISSUE THREAD

**Issue by ksblklu02** [SAP/ui5-webcomponents#9095]
### Feature Request Description

With JAWS, when user navigate with Tab key to the FileUploader component, If the corresponding label is set to "required", the component cannot announce the "required" information and the field label text.

### Proposed Solution

Adding `requried` & `accessibleName` property to the FileUploader component.

### Priority

Medium

### Privacy Policy

- [X] I’m not disclosing any internal or sensitive information.

**Comment by plamenivanov91** [comment]
Hello @SAP/ui5-webcomponents-topic-b,

Could you please take a look of this feature request?

The reporter would like to have the option to make the `FileUploader ` field `required` and to have `accesibleName`.

Regards,
Plamen Ivanov

**Comment by DMihaylova** [comment]
Hi @ksblklu02,

We already have the same requirement, so you could follow it here: https://github.com/SAP/ui5-webcomponents/issues/5893.
It's tricky to add a required property to the ui5-file-uploader as we currently see it. That's why together with design and ACC colleagues we aligned a refactoring that we plan to implement during Q3/Q4. That refactoring will also clean up the road for implementing accessibleNameRef, which we started [months ago](https://github.com/SAP/ui5-webcomponents/pull/7077/commits/10e402e470e5af253ef4ef6ae6cdb06abbc0c4e8) but we'd like to submit at the end of the whole operation in order to reduce the potential collisions and have the clearest solution as possible.

Regards,
Diana
